examples/rapids/mlflow_project/MLproject

```diff
@@ -0,0 +1,17 @@
+name: cuML RF test
+
+conda_env: envs/conda.yaml
+
+entry_points:
+    hyperopt:
+        parameters:
+            fpath: {type: str}
```

Comment:
```suggestion
```
To make fpath an optional parameter as described in [this comment](https://github.com/mlflow/mlflow/pull/3028/files#r469043050), we can just remove it from the MLproject file. Then when a user invokes `mlflow run rapids/mlflow_project -e hyperopt`, we will simply not pass the --fpath parameter to our training script, but if the user invokes `mlflow run rapids/mlflow_project -e hyperopt -P fpath=my-airline-data.parquet`, we'll still pass `--fpath my-airline-data.parquet`